src/Tribe/Customizer/Global_Elements.php

```diff
@@ -39,21 +39,53 @@ public function get_css_template( $template ) {
 			$accent_color_rgb = $accent_color::hexToRgb( $settings['accent_color'] );
 			$accent_css_rgb   = $accent_color_rgb['R'] . ',' . $accent_color_rgb['G'] . ',' . $accent_color_rgb['B'];
 
+			$accent_color_hover          = 'rgba(' . $accent_css_rgb . ',0.8)';
+			$accent_color_active         = 'rgba(' . $accent_css_rgb . ',0.9)';
+			$accent_color_background     = 'rgba(' . $accent_css_rgb . ',0.07)';
+			$accent_color_multiday       = 'rgba(' . $accent_css_rgb . ',0.24)';
```

Comment:
It would be cool to have some methods for these, or event a map for the correct opacities that we use. So that Pro can use those too.

Reply:
I agree, but out of scope for this PR.